layers/meta-tegrademo/scripts/setup-udev-rules

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+user=$(who am i | awk '{print $1}')
```

Comment:
`who am i` doesn't work on all systems (e.g., for me on Ubuntu 20.10, but I've seen this before elsewhere). I'm not sure what the right way to do this would be that handles both the normal and executed-under-sudo case.

Reply:
@dwalkes @madisongh 
You can use the following command instead:
```
user=$(id -u -n | awk '{print $1}')
```

Reply:
This didn't work (gave me "root") but I think I've got something that hopefully will be more generic, take a look at the latest one.